ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -167,48 +168,64 @@ public struct AvatarGroup: View {
+
+                        let currentAvatarHasRing = avatar.isRingVisible
+                        let nextAvatarHasRing = index + 1 < maxDisplayedAvatars ? avatars[index + 1].isRingVisible : false
+                        let avatarSizeDifference = avatarSize - nextAvatarSize
+                        let sizeDiff = !isLastDisplayed ? (currentAvatarHasRing ? avatarSizeDifference : avatarSizeDifference - ringGapOffset) :
+                            currentAvatarHasRing ? (avatarSize - ringGapOffset) - imageSize : (avatarSize - (ringGapOffset * 2)) - imageSize
+                        let x = avatarSize + tokens.interspace - ringGapOffset
+
+                        let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
+                        let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
+                        let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
+                        let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
+                        let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
+
+                        let xPosition = currentAvatarHasRing ? x - ringOuterGap - ringOuterGap : x - ringOuterGap
+                        let xPositionRTL = currentAvatarHasRing ? rtlRingPaddingInterspace : rtlNoRingPaddingInterspace
+                        let xOrigin = Locale.current.isRightToLeftLayoutDirection() ? xPositionRTL : xPosition
+                        let yOrigin = sizeDiff / 2
+                        let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
+
+                        avatarView
+                            .modifyIf(needsCutout, { view in
+                                view.mask(Avatar.AvatarCutout(
+                                            xOrigin: xOrigin,
+                                            yOrigin: yOrigin,
+                                            cutoutSize: cutoutSize)
+                                            .fill(style: FillStyle(eoFill: true)))
+                            })
+                            .padding(.trailing, tokens.style == .stack ? stackPadding : interspace)
+                            .animation(Animation.linear(duration: 0.1))
+                            .transition(AnyTransition.move(edge: .leading))
+                    }
+                    if overflowCount > 0 {
+                        createOverflow(count: overflowCount)
+                            .transition(AnyTransition.move(edge: .leading))
+                    }
+                }
+                .frame(width: geo.frame(in: .local).minX,
+                       height: geo.size.height,
+                       alignment: .leading)
             }
+            .frame(height: groupHeight)
```

Comment:
I think I added a comment about this in the past.
This is not showing in the UI of the simulator, but the Geometry Reader frame width is not being set (screenshot below).
The correct approach in this case should be calculating the width of the entire control and set it here.

![Screen Shot 2021-09-10 at 1 41 38 PM](https://user-images.githubusercontent.com/68076145/132916018-cb3c1f00-6482-4bc4-a139-3f0cba0b478b.png)